packages/patternfly-4/react-styles/README.md

```diff
@@ -105,7 +105,7 @@ const styles = StyleSheet.parse(`
 `);
 
 const Button = ({
-  variant // primary | secondary,
+  primary, // variant primary | secondary,
```

Comment:
should primary actually be "variant" here?

Reply:
fixed...